packages/flutter/test/widgets/editable_text_show_on_screen_test.dart

```diff
@@ -588,15 +623,23 @@ void main() {
         );
       }
 
-      testWidgets('focus-triggered showCaretOnScreen', (WidgetTester tester) async {
+      testWidgetsWithLeakTracking('focus-triggered showCaretOnScreen', (WidgetTester tester) async {
+        final TextEditingController textEditingController = TextEditingController();
```

Comment:
Why we stopped using textEditingController from global scope here and in tests below?